app/battle/battle.py

```diff
@@ -0,0 +1,29 @@
+from app.knights.effects import apply_effects
+from app.knights.armor_and_weapon import apply_armor_and_weapon
+from typing import Dict

```

Comment:
Add a blank line between different import groups and order them correctly to follow PEP-8